cni_test.go

```diff
@@ -374,6 +376,12 @@ func TestLibCNIType120(t *testing.T) {
 
 	err = l.Remove(ctx, "container-id1", "/proc/12345/ns/net")
 	assert.NoError(t, err)
+
+	statuses, err := l.StatusDetail(ctx)
```

Comment:
The fake configuration that is being generated, is that 1.1.0? Can you verify that I don't think the Status verb is actually being executed.